Tasks/SSH/ssh.ts

```diff
@@ -94,6 +95,18 @@ async function run() {
                 tl.debug('Copying script to remote machine.');
                 await sshHelper.copyScriptToRemoteMachine(scriptFile, scpConfig);
 
+                //change the line encodings
+                var isWin = os.type().match(/^Win/);
+                var windowsEncodedRemoteScriptPath = remoteScriptPath;
```

Comment:
Maybe rename `windowsEncodedRemoteScriptPath` to `finalRemoteScriptPath` so that the name is still applicable when the file doesn't get translated.